src/components/EditableSelectList.vue

```diff
@@ -87,6 +88,15 @@ export default {
       this.$emit('toggleCompact', !this.compact)
     },
   },
+  watch: {
+    'rows.length': {
+      handler(newLen, oldLen) {
+        if (newLen > oldLen) {
+          this.selectRow(_.maxBy(this.rows, r => +r.id));
```

Comment:
Will picking the item with the highest id always be what we want?

Reply:
I was thinking that we could end up with wonky id orders if we import libraries, but I think we actually handle that in the importer now  so it might not be an issue anymore

Reply:
👍 that sounds right to me.